integration/Tiltfile

```diff
@@ -1,5 +1,7 @@
 # -*- mode: Python -*-
 
+docker_prune_settings(disable=True)
```

Comment:
why? can you add a comment?

I would argue that we really do want to run this in the integration tests, because that's how people run tilt normally

Reply:
It's just potentially extra log output to wade through--but that's a fair point, I'll put it back